src/components/grid-aware/Hero/Hero.js

```diff
@@ -0,0 +1,93 @@
+import PropTypes from "prop-types";
+import React from "react";
+import Button from "../../inline/Button";
+import s from "./Hero.module.css";
+import Shape from "./Shape.svg";
+
+/* PropType shapes */
+
+const CTAButtonPropType = PropTypes.shape({
+  text: PropTypes.string.isRequired,
+});
+
+const HeroTextPropType = PropTypes.shape({
+  title: PropTypes.string.isRequired,
+  description: PropTypes.string.isRequired,
+});
+
+/* Subcomponents */
+
+const CTAButtons = ({ buttons }) => (
+  <div className={s.ctaButtonBlock}>
+    {buttons.map((button) => (
+      <Button key={button.text}>{button.text}</Button>
+    ))}
+  </div>
+);
+
+CTAButtons.propTypes = {
+  buttons: PropTypes.arrayOf(CTAButtonPropType).isRequired,
+};
+
+const HeroText = ({ text }) => (
+  <div className={s.textContainer}>
+    <div className={s.title}>{text.title}</div>
+    <div className={s.description}>{text.description}</div>
+  </div>
+);
+
+HeroText.propTypes = {
+  text: PropTypes.arrayOf(HeroTextPropType).isRequired,
+};
+
+const PlayButton = ({ link }) => (
+  <div className={s.playButtonContainer}>
+    <a rel="noreferrer" href={link} target="_blank">
+      <img src={Shape} alt="Play Button" className={s.playButton} />
+    </a>
+  </div>
+);
+
+PlayButton.propTypes = {
+  link: PropTypes.string.isRequired,
+};
+
+/* Main component */
+
+const Hero = ({ ctaButtons, heroImage, heroLink, heroText }) => {
+  const GridAreaLeft = () => (
+    <div className={s.gridAreaLeft}>
+      <HeroText text={heroText} />
+      <CTAButtons buttons={ctaButtons} />
+    </div>
+  );
```

Comment:
I think this is probably short enough that you could just inline this directly into the main `return` statement below. In https://github.com/ShelterTechSF/sheltertech.org/pull/66, I had split it out because that component was large and complex enough that I felt that splitting it up made it easier to follow. This component has a lot less stuff, so I think it suffices to keep it simple.